Ginger/GingerCoreNET/Reports/Ginger-Web-Client/assets/sass/layout/_dashboard.scss

```
@@ -40,7 +40,7 @@ $salesRowTextColor: #212121;
 $salesUpIconColor: #4caf50;
 $salesDownIconColor: #f44336;
 
-/* Time-line */
+/*Timeline*/
```

Stylelint (17.6.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)